src/libraries/System.Private.CoreLib/src/System/Reflection/ConstructorInvoker.cs

```diff
@@ -11,6 +11,17 @@
 
 namespace System.Reflection
 {
+    /// <summary>
+    /// Invokes the method reflected by the provided <see cref="ConstructorInfo"/>.
```

Comment:
NIT: Not sure if it was intentional but some of the mentions of `method` in this file might better to be `constructor` instead
```suggestion
    /// Invokes the constructor reflected by the provided <see cref="ConstructorInfo"/>.
```